### PR TITLE
Always use npm in test app script

### PR DESCRIPTION
**Pull request by matteodepalo**
Testing with npm will help us catch bugs faster.

### CODE REVIEW

File: bin/create-test-app.js [hunk 1]
```diff
@@ -80,11 +80,7 @@ program
...
         break;
       case undefined:
-        if (options.install === "local") {
-          nodePackageManager = "pnpm";
-        } else if (options.install === "nightly") {
-          nodePackageManager = "npm";
-        }
+        nodePackageManager = "npm";
         break;
       default:
```
Comment (gonzaloriestra) [inline bin/create-test-app.js:85]: We can just set npm in the default section and forget about undefined, right?

Comment (matteodepalo) [reply bin/create-test-app.js:85]: If we don't add an undefined case then we can't omit the flag. Or did you have another fix in mind? 
I added `    let nodePackageManager = "npm";` and removed the undefined case.

Comment (gonzaloriestra) [reply bin/create-test-app.js:85]: Maybe I'm missing something, but my idea was:

```
switch (options.packageManager) {
  case "yarn":
  case "pnpm":
    nodePackageManager = options.packageManager;
    break;
  default:
    nodePackageManager = "npm";
}
```

Comment (gonzaloriestra) [reply bin/create-test-app.js:85]: But this is an internal script, I think it's fine and the code is simpler. But just an opinion.

**Pull request merged**